docs: deployment.md error

<!-- This form is for bug reports ONLY!

If you're looking for a help then check our [Slack Channel](https://cloud-native.slack.com/messages/volcano) or have a look at our [dev mailing](https://groups.google.com/forum/#!forum/volcano-sh) 
-->

**What happened**:

https://github.com/volcano-sh/volcano/blob/master/docs/development/development.md

Make some local change of the code, then build vc-controller-manager
```
make image.vc-controller-manager
```
Local result:
```
➜  volcano git:(remove-vendor) make image.vc-controller-manager
make: *** No rule to make target `image.vc-controller-manager'.  Stop.
```


**What you expected to happen**:
I think the right command is 
```
make vc-controller-manager
```

**How to reproduce it (as minimally and precisely as possible)**:


**Anything else we need to know?**:

**Environment**:
- Volcano Version:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):mac m1
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/docs/development/development.md
+++ b/docs/development/development.md
@@ -58,7 +58,7 @@ make images DOCKER_PLATFORMS="linux/amd64,linux/arm64" BUILDX_OUTPUT_TYPE=regist
 ```
 
 
-## Building a specific docker image
+## Building a specific component
 
 If you want to make a local change and test some component, say `vc-controller-manager`, you
 could do:
@@ -83,7 +83,7 @@ export TAG=citadel
 Make some local change of the code, then build `vc-controller-manager`
 
 ```bash
-make image.vc-controller-manager
+make vc-controller-manager
 ```
 
 ## Building the Volcano manifests

